### PR TITLE
{lib}[GCCcore/13.3.0] aiohttp v3.10.10

### PR DESCRIPTION
**Pull request by lcniel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/aiohttp/aiohttp-3.10.10-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }),
+    ('aiosignal', '1.3.1', {
+        'checksums': ['54cd96e15e1649b75d6c87526a6ff0b6c1b0dd3459f43d9ca11d48c339b68cfc'],
+    }),
+    ('aiohappyeyeballs', '2.3.2', {
+        'checksums': ['77e15a733090547a1f5369a1287ddfc944bd30df0eb8993f585259c34b405f4e'],
+    }),
+    (name, version, {
+        'checksums': ['0631dd7c9f0822cc61c88586ca76d5b5ada26538097d0f1df510b082bad3411a'],
+    }),
```
Comment (Micket) [inline easybuild/easyconfigs/a/aiohttp/aiohttp-3.10.10-GCCcore-13.3.0.eb:49]: please update
multidict 6.1.0
yarl 1.15.4
aiohappyeyeballs 2.4.3

**Comment by lcniel** [comment]
Test report by @lcniel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera02-1 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/63adcb623bb7e47355155f971227c709 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera30-1 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/6dd127802ac85de3408f89b7c276d203 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera22-2 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/eb7719692506578069bc4566b79d13a3 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera30-3 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/a860145d32a022ff942068a64067e6dd for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera22-2 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/139b5dfddcad23ab938377b11a1eed04 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera22-2 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/65874dd9fb2bfe7ca567bc1c6547f4f2 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera22-2 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/68386f07ea2d1f97ea179735efa87b9d for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Pull request merged**